src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -479,4 +479,7 @@ For details on valid range queries, please refer to Search Matching section in C
     <value>Expected value '{0}' to be one of the following values: [{1}]</value>
     <comment>{0} Actual value. {1} Comma-separated list of expected values.</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="WorkitemInstanceAlreadyExists" xml:space="preserve">
+    <value>The workitem instance already exists.</value>
```

Comment:
should we add the UID in the message and mention we do not support workitems with same UID and ask them to retry with a unique one?

Reply:
Added